ansible/triliovault-sub-tasks/add_mount_fuse.yml

```diff
@@ -0,0 +1,37 @@
+--- 
+
+- name: Check "/var/lib/lxc/{{ container_name }}/autodev" is Present OR Not
+  stat: path="/var/lib/lxc/{{ container_name }}/autodev"
+  register: chk_autodev
+  delegate_to: "{{ physical_host }}"
+
+- name: Add 'cp -dpR /dev/fuse ${LXC_ROOTFS_MOUNT}/dev' to autodev
+  lineinfile:
+    path: "/var/lib/lxc/{{ container_name }}/autodev"
+    line: cp -dpR /dev/fuse ${LXC_ROOTFS_MOUNT}/dev
+    create: no
+    state: present 
+  register: reg_1
+  delegate_to: "{{ physical_host }}"
+  when:
+    - chk_autodev.stat.exists == true
+
+- name: print output
+  debug:
+    msg: "{{ reg_1 }}"
+
+- name: Add 'chmod 0666 ${LXC_ROOTFS_MOUNT}/dev/fuse' to autodev
+  lineinfile:
+    path: "/var/lib/lxc/{{ container_name }}/autodev"
+    line: chmod 0666 ${LXC_ROOTFS_MOUNT}/dev/fuse
```

Comment:
Are we changing permissions of /dev/fuse on host?

Reply:
Yes that's true

Reply:
we need the right access on this device for the **nova** user, which needs **rw** access for other users.

the device is owned by the root user and for another user it's read-only.
we need **rw** for nova user, and this change gets affected only for our container, it does not get reflected on the host